posts/2024-09-05-impl-trait-capture-rules.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: post
+title: "Changes to impl Trait in Rust 2024"
+author: Niko Matsakis
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+This blog post describes some small but significant changes with (return position) `impl Trait` capture rules that are coming in Rust 2024. The goal of these changes is to simplify how `impl Trait` works to better match what people want while also giving a flexible syntax that allows users to have full control when needed.
+
+## TL;DR
+
+* Return-position impl Trait in Rust 2024 can now reference all fn arguments by default.
```

Comment:
we should link to the RFCs and issues where decisions were made here I think so people can click for more details